ansible/roles/destroy-vm/tasks/main.yml

```diff
@@ -15,3 +15,7 @@
 - name: Delete IBM Z VM
   include_tasks: destroy-s390x-vm.yml
   when: arch_info == "s390x"
+
+- name: Delete IBM POWER VSI
+  include_tasks: destroy-ppc64le-vm.yml
```

Comment:
What is the difference between destroying a Power VM and destroying a Z VM? Do we need a separate task file for each one?

Reply:
To be a bit clearer with this one - is the process between the two similar enough that we can do it in the same way for each?

Reply:
We cant use same playbook for both because variables are different Also Z creation and destroy  in VPC